modeltranslation/manager.py

```diff
@@ -414,8 +414,9 @@ def raw_values(self, *fields):
     def _values(self, *original, **kwargs):
         if not kwargs.pop('prepare', False):
             return super(MultilingualQuerySet, self)._values(*original, **kwargs)
+        selects_all = kwargs.pop('selects_all', False)
```

Comment:
i think `selects_all` should be defined explicitly in function definition, becouse otherwise it could be handed over to super()

Reply:
or, at least, this line should be above 415, if we want to match original definition.